app/controllers/api/v3/stream_daily_averages_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V3
+    class StreamDailyAveragesController < BaseController
+      def show
```

Comment:
Maybe we could use interactor here and move into direction of using `Result` to handle success/failure scenarios. WDYT?